### DISCUSSION TITLE
How to enable pop-ups for errors?

### DISCUSSION THREAD

**Discussion by wheinen** [dbeaver/dbeaver#35460] [Q&A] [answered]
When I execute a wrong query or a query that returns an error, I don't get a pop-up notification, or any clear message at all.
The only thing I see is an exclamation mark on the line that gives the error (which I have to mouse-over to see the actual error), and a small error on the bottom of the results tab, like here:
![image](https://github.com/user-attachments/assets/15a67768-4535-41d7-a28d-576b6ef3c9af)

When my colleagues execute a wrong query, they get a pop-up message or a largely visible message in the results panel, so it's immediately clear something went wrong.

I've searched and searched the settings menu, but I can't seem to find how to enable this. 
Does anybody have the answer for me?

**Comment by tlm365** [comment]
Looks like you are using an outdated version of DBeaver. Consider updating to the latest version.

**Comment by wheinen** [comment]
I'm on the latest version; 24.2.0

Also, on the 'Data Editor' settings page, I have 'Show errors in a dialog' enabled. Still, it behaves like in the screenshot.

**Comment by wheinen** [comment, marked as answer]
Nearly 2 years later, I finally found the answer thanks to Gemini... 
Go to Window > Preferences > Editors > SQL Editor > SQL Processing and set "Error handling" to "Stop + rollback" instead of "Ignore"
<img width="835" height="768" alt="image" src="https://github.com/user-attachments/assets/ca5b3a27-de50-4dde-a712-54f05bb1d88f" />
Apparently if you want error pop ups, this is necessary. Stupid if you ask me, because I'd like error pop-ups without rollbacking, but well...
